### ISSUE TITLE
why 3 trained models for one problem

### ISSUE THREAD

**Issue by mamonraab** [swathikirans/violence-recognition-AVSS#4]
hi , first can you explain why you did 3 times training  for your model  isn't the goal is one model that can do good in all datasets coz we have one problem space is a violent problem 
also how you can say for sure your models is not overfitted and you just reported the test result  isn't the best way to show us the training vs test loss also the samples number so small and you make 100 epochs this for sure  will lead to overfitting

**Comment by swathikirans** [comment]
Hi, there were three standard datasets for violence recognition problem and I wanted to report the performance of the proposed model on these three datasets. The three models given in the repo are the models that were trained on these three datasets.

Coming to the overfitting problem, overfitting is not undesirable all the time. Since I am getting a test performance of more than 90%, there is no reason to worry about overfitting. Overfitting become undesirable only when the test performance is low compared to the training performance.

**Comment by mamonraab** [comment]
now if want to use the model for real-world problem which of the 3 I must use ? said i have no data and don't want to do training

in the paper do you report the test accuracy or the training accuracy and  can i test the model on different dataset get from the  youtube

**Comment by swathikirans** [comment]
It depends on what kind of videos you are going to work on. Each of the three datasets have their own characteristics. Out of the three datasets, violentflows dataset represents real world videos the most.

The paper reports test accuracy and you are free to evaluate the pre-trained models on whatever datasets you like to.

**Comment by mamonraab** [comment]
thanks alot  , i will try its on some youtube videos
